text_mining.py

```diff
@@ -0,0 +1,131 @@
+import requests
+import nltk
+import operator
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+from collections import Counter
+
+
+papers = ['fedpaper1.txt', 'fedpaper2.txt', 'fedpaper3.txt', 'fedpaper4.txt', 'fedpaper5.txt', 'fedpaper6.txt', 'fedpaper7.txt', 'fedpaper8.txt', 'fedpaper9.txt', 'fedpaper10.txt']
+
+
+def fed_paper_creator():
+    '''
+    This function writes the lines from each one of the first 10 Federalist Papers to a corresponding
+    text file. It makes the words into one string, and makes them all lowercase
+    '''
+    input_file = open("fedpapers.txt", "r")
+    lines = input_file.readlines()
+
+    paper1 = str.lower(' '.join(lines[32:194]))
+    paper2 = str.lower(' '.join(lines[196:370]))
+    paper3 = str.lower(' '.join(lines[372:535]))
+    paper4 = str.lower(' '.join(lines[537:708]))
+    paper5 = str.lower(' '.join(lines[710:857]))
+    paper6 = str.lower(' '.join(lines[859:1108]))
+    paper7 = str.lower(' '.join(lines[1110:1339]))
+    paper8 = str.lower(' '.join(lines[1341:1551]))
+    paper9 = str.lower(' '.join(lines[1553:1762]))
+    paper10 = str.lower(' '.join(lines[1764:2064]))
+
+    lines = [paper1, paper2, paper3, paper4, paper5, paper6, paper7, paper8, paper9, paper10]
+    papers = ['fedpaper1.txt', 'fedpaper2.txt', 'fedpaper3.txt', 'fedpaper4.txt', 'fedpaper5.txt', 'fedpaper6.txt', 'fedpaper7.txt', 'fedpaper8.txt', 'fedpaper9.txt', 'fedpaper10.txt']
+
+    for i in range(len(lines)):
+        file = open(papers[i], 'w')
+        file.write(lines[i])
+        file.close()
+
+
+def paper_split(paper):
+    '''
+    This function tokenizes each one of the Federalist Papers. It removes punctuation, spaces, and about 25 of the most common words according to Wikipedia.
+    '''
+    tokenpaper = open(paper).read()
+    words = nltk.word_tokenize(tokenpaper)
+    '''common_words = ['the', 'be', 'to', 'of', 'and', 'a', 'in', 'that', 'have', 'it', 'for', 'not', 'on', 'with', 'he', 'as', 'do']
+    common_words2 = ['at', 'this', 'but', 'by', 'from']'''
+    for word in words:
+        if word == '.' or word == ';' or word == ':' or word == ',' or word == '-' or word == '!' or word == '"' or word == '?' or word == '(' or word == ')':
+            words.remove(word)
+        if word == "'" or word == '[' or word == ']' or word == "'\\n" or word == '--':
+            words.remove(word)
+        '''if word in common_words or word in common_words2:
+            words.remove(word)
+        if word == 'the' or word == 'a':
+            words.remove(word)'''
+    return words
+    print(words)
+
+
+def new_paper(papers):
+    '''
+    This function writes the new tokenized words into the old text file.
+    '''
+    fed_paper_creator()
+    for paper in papers:
+        words = paper_split(paper)
+        wordstr = str(words)
+        file = open(paper, 'w')
+        file.write(wordstr)
+        file.close()
+        return words
+
```

Comment:
I think this is an incorrect placement of return statement. In current implementation, for loop will only iterate through first paper and return the words in the first paper. Remember that return statement immediately terminates a function, regardless of whether it is a loop.